packages/terra-button/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Changelog
 Unreleased
 ----------
 
+1.13.1 - (November 21, 2017)
+------------------
+### Changed
+* Corrected typo in `terra-button` comment.
```

Comment:
maybe not terra-button, rather typo in 'css' comment so it matches on where to look

Reply:
Fair point, I've updated the message.